Library/Styles/LoginStyles.swift

```diff
@@ -100,6 +100,17 @@ public let loginRootStackViewStyle =
         : .init(topBottom: Styles.grid(2), leftRight: Styles.grid(3))
 }
 
+public let facebookConfirmationStackViewStyle =
+  UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+    <> UIStackView.lens.layoutMargins %~~ { _, stack in
```

Comment:
May not be worth the layered logic, but we could reuse `loginRootStackViewStyle` and ensure that the base insets don't go out of sync:

``` swift
public let facebookConfirmationStackViewStyle =
  loginRootStackViewStyle
    <> UIStackView.lens.layoutMargins..UIEdgeInsets.lens.topBottom %~~ { topBottom, stack in
      !stack.traitCollection.isRegularRegular && stack.traitCollection.isVerticallyCompact
        ? Styles.grid(2)
        : topBottom
}
```